manuscript/our_first_game.md

```diff
@@ -99,34 +98,27 @@ a couple more small steps to take before we can see it rendered in the browser.
 
 First, we'll need to update our `brunch-config.js` file so that both `Main.elm`
 and `Game.elm` will both be compiled (note that the only change here is on the
-`elmBrunch` line).
+`mainModules` line).
 
 ```javascript
-exports.config = {
-  files: {
-    javascripts: { joinTo: "js/app.js" },
-    stylesheets: { joinTo: "css/app.css" },
-    templates: { joinTo: "js/app.js" }
-  },
-  conventions: { assets: /^(static)/ },
-  paths: {
-    watched: ["../lib/platform/web/elm", "static", "css", "js", "vendor"],
-    public: "../priv/static"
-  },
   plugins: {
-    babel: { ignore: [/vendor/] },
-    elmBrunch: { elmFolder: "../lib/platform/web/elm", mainModules: ["Main.elm", "Game.elm"], outputFolder: "../../../../assets/vendor" }
+    babel: {
+      ignore: [/vendor/]
+    },
+    elmBrunch: {
+      mainModules: ["elm/Main.elm", "elm/Game.elm"],
+      makeParameters: ["--debug"],
+      outputFolder: "../assets/js"
+    }
   },
-  modules: { autoRequire: { "js/app.js": ["js/app"] } },
-  npm: { enabled: true }
-};
 ```
 
 Lastly, we can update our `app.js` file to render our game inside the Phoenix
 application. At the bottom of our `assets/js/app.js` file, let's update our
 code to look like this:
 
 ```javascript
+import ElmGame from "./game"
```

Comment:
I have no idea if this is right or how we should be doing it, but this is how I got it to work.